src/components/Controls/SnackChatButton/SnackChatButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+
+import ChatBubble from '@material-ui/icons/ChatBubble';
+import ChatInput from './ChatInput';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Fab from '@material-ui/core/Fab';
+import Tooltip from '@material-ui/core/Tooltip';
+
+import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
+import { useSnackbar } from 'notistack';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    fab: {
+      margin: theme.spacing(1),
+    },
+  })
+);
+
+export default function SnackChatButton() {
+  const classes = useStyles();
+  const { room } = useVideoContext();
+  const { enqueueSnackbar } = useSnackbar();
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleSend = (message: string) => {
+    const [dataTrackPublication] = [...room.localParticipant.dataTracks.values()];
+    if (dataTrackPublication) {
+      const fullMessage = `${room.localParticipant.identity} says: ${message}`;
+      dataTrackPublication.track.send(fullMessage);
+      enqueueSnackbar(fullMessage);
+    }
+  };
+
+  return (
+    <ClickAwayListener onClickAway={() => setIsOpen(false)}>
+      <Tooltip
+        title={<ChatInput onSend={handleSend} />}
```

Comment:
I think it goes here! 

But a title tool tip saying "Chat" when you hover over the SnackChat button would be useful! Then when you click on it, it opens the ChatInput component? What do you think?